python/lsst/jointcal/jointcal.py

```diff
@@ -135,7 +156,8 @@ def _build_ccdImage(self, dataRef, associations, jointcalControl):
 
         goodSrc = self.sourceSelector.selectSources(src)
 
-        # TODO: NOTE: Leaving the old catalog and comparison code in while we
+        # ------------------
+        # TODO: Leaving the old catalog and comparison code in while we
```

Comment:
I just submitted an RFC. I'll note that I couldn't find any explicit statement about this in the python coding standards.